### ISSUE TITLE
report a bot

### ISSUE THREAD

**Issue by TheMoksej** [discordextremelist/website#159]
A button on the bots/servers/templates page which says "report". When bot/server/template gets reported assistants and administrators can see that somewhere in the staff page.

**Comment by advaith1** [comment]
I've considered this but I don't think it would really be worth it to build a system for this, as people can just report it via the server. Maybe we could add a button and input to surface it more but just have it send to a staff-only channel in the server?

**Comment by TheMoksej** [comment]
> I've considered this in the past but I don't think it would really be worth it to build a system for this, as people can just report it via the server. Maybe we could add a button and input to surface it more but just have it send to a staff-only channel in the server?

I get your point. And yes sending it to a staff-only channel works as well, that's what I've originally thought about before submitting this suggestion, but I'm not sure why I changed my mind while submitting it :/

**Comment by Cairo2k18** [comment]
> I've considered this in the past but I don't think it would really be worth it to build a system for this, as people can just report it via the server. Maybe we could add a button and input to surface it more but just have it send to a staff-only channel in the server?

Better than creating a whole interface for it, allow users to report with a short reason?

**Comment by Cairo2k18** [comment]
oops

**Comment by advaith1** [comment]
yeah, i was thinking the button would require login and go to a popup or page with a reason input, then throw the reporter, reason, and bot page link into #reports or smth

**Comment by Cairo2k18** [comment]
Thoughts on this being sent to a channel?
Would also be saved in del audit log, and the author text links to the bot's listing.

![image](https://user-images.githubusercontent.com/38713645/96709263-f0eeda80-13f6-11eb-8cd9-4bf4067b0887.png)

**Comment by TheMoksej** [comment]
looks good, I'd probably chabge the color to a bit darker. That looks too light, but that might just be the image.

**Comment by Cairo2k18** [comment]
> looks good, I'd probably change the color to a bit darker. That looks too light, but that might just be the image.

If it's too dark it'll look disgusting

**Comment by carolinaisslaying** [comment]
Idea is that this will be implemented using a modal with some preset reasons (incl other), an option for additional information, and send a report message to the #site-alerts channel. Would require user authentication.
